src/server/middleware/executor/worker/node_worker.classes.build.js

```diff
@@ -3640,7 +3699,8 @@ define('blob/BlobClient',[
                 hashes[filename] = hash;
 
                 if (err) {
-                    error += 'putFile error: ' + err.toString();
+                    error = err;
```

Comment:
It will return only with the last error, but that is fine, because we log all errors.